torchci/scripts/no_flaky_tests_alert.py

```diff
@@ -0,0 +1,92 @@
+import argparse
+import json
+import os
+from datetime import datetime, timedelta
+from typing import Any, Dict, List, Optional
+
+import requests
+from check_alerts import (
+    GRAPHQL_URL,
+    HEADERS,
+    PYTORCH_REPO_NAME,
+    REPO_OWNER,
+    create_issue,
+)
+
+FLAKY_TESTS_LABEL = "module: flaky-tests"
+NO_FLAKY_TESTS_LABEL = "no-flaky-tests-alert"
+FLAKY_TESTS_SEARCH_PERIOD_DAYS = 7
+
+NUM_ISSUES_QUERY = """
+query ($query: String!) {
+  search(type: ISSUE, query: $query) {
+    issueCount
+  }
+}
+"""
+
+def get_num_issues_with_label(owner: str, repo: str, label: str, from_date: Optional[str]= None) -> int:
+    query = f'repo:{owner}/{repo} label:"{label}" is:issue is:open'
+    if from_date:
+        query += f" created:>={from_date} "
+    try:
+        r = requests.post(
+            GRAPHQL_URL,
+            json={"query": NUM_ISSUES_QUERY, "variables": {"query": query}},
+            headers=HEADERS,
+        )
+        r.raise_for_status()
+        data = json.loads(r.text)
+        return data["data"]["search"]["issueCount"]
+    except Exception as e:
+        raise RuntimeError("Error fetching issues count", e)
+
+
+def handle_flaky_tests_alert(dry_run: bool) -> Optional[Dict[str, Any]]:
+    from_date = (
+        datetime.today() - timedelta(days=FLAKY_TESTS_SEARCH_PERIOD_DAYS)
+    ).strftime("%Y-%m-%d")
+    num_issues_with_flaky_tests_label = get_num_issues_with_label(
+        REPO_OWNER, PYTORCH_REPO_NAME, FLAKY_TESTS_LABEL, from_date
+    )
+    print(
+        f"Num issues with `{FLAKY_TESTS_LABEL}` label: {num_issues_with_flaky_tests_label}",
+    )
+    if num_issues_with_flaky_tests_label == 0:
+        return create_issue(generate_no_flaky_tests_issue(), dry_run)
+
+
+def generate_no_flaky_tests_issue() -> Any:
```

Comment:
Do you think it's a good idea to have an upper bound alert as well to capture the case in which there are many more flaky tests created than usual?  Specifically, the number of disabled tests created during FLAKY_TESTS_SEARCH_PERIOD_DAYS is between the lower bound of 0 and the upper bound of N (TBD)